react/features/base/connection/actions.native.js

```diff
@@ -322,7 +324,20 @@ export function disconnect() {
 
             promise
                 = conference_.leave()
-                    .catch(() => {
+                    .then(() => {
+                        // If the conference is still in the leaving flag it
+                        // means that for some reason lib-jitsi-meet has not
+                        // dispatched the left event.
+                        const { leaving }
+                            = getState()['features/base/conference'];
+
+                        if (leaving === conference_) {
+                            dispatch(conferenceLeft(conference_));
+                            logger.warn(
```

Comment:
I don't get the idea of this warn here. So you saw that lib-jitsi-meet doesn't emit the CONFERENCE_LEFT event, we chose to fix it in jitsi-meet, you chose to not fix it in lib-jitsi-meet, and now you're going to warn in jitsi-meet that lib-jitsi-meet didn't do it!? It's already stated in the comment that that's what happened, it's the whole reason why this additional source code exists in jitsi-meet...

Reply:
But I can have it removed if you want